File: model/Default.vb
Public Class [Default]
    Public Name As String
    Public Value As String

    Public Sub New(ByVal name As String, ByVal value As String)
        Me.Name = name
        Me.Value = value
    End Sub

    Public Function Script() As String
        Return String.Format("CONSTRAINT {0} DEFAULT {1}", Name, Value)
    End Function
End Class

File: model/Default.cs
namespace model {
	public class Default {
		public string Name;
		public string Value;

		public Default(string name, string value) {
			this.Name = name;
			this.Value = value;
		}

		public string Script() {
			return string.Format("CONSTRAINT [{0}] DEFAULT {1}", this.Name, this.Value);
		}
	}
}